README.md

```diff
@@ -1,6 +1,6 @@
 # Pop CLI
 
-<img src=".icons/logo.jpeg"></img>
+<img src="/.icons/logo.jpeg"></img>
```

Comment:
This doesn't appear to be working correctly. I can still not see the banner.

Reply:
This make the logo appears in pop-cli/crate/README.md